packages/types/index.d.ts

```diff
@@ -408,6 +408,8 @@ export interface IApp {
   connections?: IConnection[]
   description?: string
   isNewApp?: boolean
+  connectionStepLabel?: string // for step accordion label
+  settingsStepLabel?: string // for step accordion label: app level
```

Comment:
nit: how about this so that we group everything together:
```suggestion
  substepLabels: {
    connectionSubstep: string
    setupSubsetup: string
  }
```